include/RED4ext/RTTISystem-inl.hpp

```diff
@@ -20,12 +20,17 @@ RED4EXT_INLINE void RED4ext::CRTTISystem::RegisterType(CBaseRTTIType* aType)
 
 RED4EXT_INLINE void RED4ext::RTTIRegistrator::Add(CallbackFunc aRegFunc, CallbackFunc aPostRegFunc, bool aUnused)
```

Comment:
Should we deprecate this?

Reply:
Yeah. `[[deprecated("Use 'AddRegisterCallback()' and `AddPostRegisterCallback()` instead.")]]` or something more verbose?

Reply:
That is perfect.